src/index.js

```diff
@@ -91,8 +91,12 @@ const generateEDL = ({
 	fileNames,
 	sampleRate,
 	frameRate,
-	projectName
+	projectName,
+	adlUid,
+	projectCreatedDate
 }) => {
+	const ADLUid = adlUid? adlUid: str(uuid1());
```

Comment:
same here. ADLUid = adlUid? Looks messy. Is it possible to use like `id`, `uid`, or something like `audioId` or `outputId`. Just spitballin' 🤷‍♂

Reply:
yeah, the parameter is `adlUid` but wanted to provide a default, so added `ADLUid` etc..

💡 going to try and see if it can be cleaned up with default params